src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double length;
+    private double width;
+
+    public Rectangle(String color, double length, double width) {
+        super(color);
+        this.length = length;
+        this.width = width;
+    }
+
+    @Override
+    public double getArea() {
+        return length * width;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rectangle, area: "
```

Comment:
Avoid using `System.out.println()` for drawing the figure. It would be better to return the drawing information as a String which can be printed elsewhere, keeping the method's responsibility focused on building the representation of the object.